[2014-08-24 00:52:41] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /" (uncaught exception) at /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 145 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /\" at /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php:145, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at /home/malcolm/silox/vendor/symfony/routing/Symfony/Component/Routing/Matcher/UrlMatcher.php:96)"} []
[2014-08-24 00:52:41] app.ERROR: No route found for "GET /" [] []
[2014-08-24 00:52:41] app.ERROR: #0 [internal function]: Symfony\Component\HttpKernel\EventListener\RouterListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\EventDispatcher)) #1 /home/malcolm/silox/vendor/symfony/event-dispatcher/Symfony/Component/EventDispatcher/EventDispatcher.php(164): call_user_func(Array, Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\EventDispatcher)) #2 /home/malcolm/silox/vendor/symfony/event-dispatcher/Symfony/Component/EventDispatcher/EventDispatcher.php(53): Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #3 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(126): Symfony\Component\EventDispatcher\EventDispatcher->dispatch('kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #4 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(66): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #5 /home/malcolm/silox/vendor/silex/silex/src/Silex/Application.php(538): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #6 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(466): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #7 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(429): Symfony\Component\HttpKernel\HttpCache\HttpCache->forward(Object(Symfony\Component\HttpFoundation\Request), true) #8 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(329): Symfony\Component\HttpKernel\HttpCache\HttpCache->fetch(Object(Symfony\Component\HttpFoundation\Request), true) #9 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(193): Symfony\Component\HttpKernel\HttpCache\HttpCache->lookup(Object(Symfony\Component\HttpFoundation\Request), true) #10 /home/malcolm/silox/vendor/silex/silex/src/Silex/HttpCache.php(35): Symfony\Component\HttpKernel\HttpCache\HttpCache->handle(Object(Symfony\Component\HttpFoundation\Request)) #11 /home/malcolm/silox/web/index.php(10): Silex\HttpCache->run() #12 {main} [] []
[2014-08-24 00:52:41] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /favicon.ico" (uncaught exception) at /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 145 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /favicon.ico\" at /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php:145, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at /home/malcolm/silox/vendor/symfony/routing/Symfony/Component/Routing/Matcher/UrlMatcher.php:96)"} []
[2014-08-24 00:52:41] app.ERROR: No route found for "GET /favicon.ico" [] []
[2014-08-24 00:52:41] app.ERROR: #0 [internal function]: Symfony\Component\HttpKernel\EventListener\RouterListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\EventDispatcher)) #1 /home/malcolm/silox/vendor/symfony/event-dispatcher/Symfony/Component/EventDispatcher/EventDispatcher.php(164): call_user_func(Array, Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\EventDispatcher)) #2 /home/malcolm/silox/vendor/symfony/event-dispatcher/Symfony/Component/EventDispatcher/EventDispatcher.php(53): Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #3 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(126): Symfony\Component\EventDispatcher\EventDispatcher->dispatch('kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #4 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(66): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #5 /home/malcolm/silox/vendor/silex/silex/src/Silex/Application.php(538): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #6 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(466): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #7 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(429): Symfony\Component\HttpKernel\HttpCache\HttpCache->forward(Object(Symfony\Component\HttpFoundation\Request), true) #8 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(329): Symfony\Component\HttpKernel\HttpCache\HttpCache->fetch(Object(Symfony\Component\HttpFoundation\Request), true) #9 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(193): Symfony\Component\HttpKernel\HttpCache\HttpCache->lookup(Object(Symfony\Component\HttpFoundation\Request), true) #10 /home/malcolm/silox/vendor/silex/silex/src/Silex/HttpCache.php(35): Symfony\Component\HttpKernel\HttpCache\HttpCache->handle(Object(Symfony\Component\HttpFoundation\Request)) #11 /home/malcolm/silox/web/index.php(10): Silex\HttpCache->run() #12 {main} [] []
[2014-08-24 00:53:52] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /api" (uncaught exception) at /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 145 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /api\" at /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php:145, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at /home/malcolm/silox/vendor/symfony/routing/Symfony/Component/Routing/Matcher/UrlMatcher.php:96)"} []
[2014-08-24 00:53:52] app.ERROR: No route found for "GET /api" [] []
[2014-08-24 00:53:52] app.ERROR: #0 [internal function]: Symfony\Component\HttpKernel\EventListener\RouterListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\EventDispatcher)) #1 /home/malcolm/silox/vendor/symfony/event-dispatcher/Symfony/Component/EventDispatcher/EventDispatcher.php(164): call_user_func(Array, Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\EventDispatcher)) #2 /home/malcolm/silox/vendor/symfony/event-dispatcher/Symfony/Component/EventDispatcher/EventDispatcher.php(53): Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #3 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(126): Symfony\Component\EventDispatcher\EventDispatcher->dispatch('kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #4 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(66): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #5 /home/malcolm/silox/vendor/silex/silex/src/Silex/Application.php(538): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #6 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(466): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #7 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(429): Symfony\Component\HttpKernel\HttpCache\HttpCache->forward(Object(Symfony\Component\HttpFoundation\Request), true) #8 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(329): Symfony\Component\HttpKernel\HttpCache\HttpCache->fetch(Object(Symfony\Component\HttpFoundation\Request), true) #9 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(193): Symfony\Component\HttpKernel\HttpCache\HttpCache->lookup(Object(Symfony\Component\HttpFoundation\Request), true) #10 /home/malcolm/silox/vendor/silex/silex/src/Silex/HttpCache.php(35): Symfony\Component\HttpKernel\HttpCache\HttpCache->handle(Object(Symfony\Component\HttpFoundation\Request)) #11 /home/malcolm/silox/web/index.php(10): Silex\HttpCache->run() #12 {main} [] []
[2014-08-24 00:53:58] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /siloname" (uncaught exception) at /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 145 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /siloname\" at /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php:145, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at /home/malcolm/silox/vendor/symfony/routing/Symfony/Component/Routing/Matcher/UrlMatcher.php:96)"} []
[2014-08-24 00:53:58] app.ERROR: No route found for "GET /siloname" [] []
[2014-08-24 00:53:58] app.ERROR: #0 [internal function]: Symfony\Component\HttpKernel\EventListener\RouterListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\EventDispatcher)) #1 /home/malcolm/silox/vendor/symfony/event-dispatcher/Symfony/Component/EventDispatcher/EventDispatcher.php(164): call_user_func(Array, Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\EventDispatcher)) #2 /home/malcolm/silox/vendor/symfony/event-dispatcher/Symfony/Component/EventDispatcher/EventDispatcher.php(53): Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #3 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(126): Symfony\Component\EventDispatcher\EventDispatcher->dispatch('kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #4 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(66): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #5 /home/malcolm/silox/vendor/silex/silex/src/Silex/Application.php(538): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #6 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(466): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #7 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(429): Symfony\Component\HttpKernel\HttpCache\HttpCache->forward(Object(Symfony\Component\HttpFoundation\Request), true) #8 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(329): Symfony\Component\HttpKernel\HttpCache\HttpCache->fetch(Object(Symfony\Component\HttpFoundation\Request), true) #9 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(193): Symfony\Component\HttpKernel\HttpCache\HttpCache->lookup(Object(Symfony\Component\HttpFoundation\Request), true) #10 /home/malcolm/silox/vendor/silex/silex/src/Silex/HttpCache.php(35): Symfony\Component\HttpKernel\HttpCache\HttpCache->handle(Object(Symfony\Component\HttpFoundation\Request)) #11 /home/malcolm/silox/web/index.php(10): Silex\HttpCache->run() #12 {main} [] []
[2014-08-24 00:54:53] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /v1/" (uncaught exception) at /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 145 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /v1/\" at /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php:145, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at /home/malcolm/silox/vendor/symfony/routing/Symfony/Component/Routing/Matcher/UrlMatcher.php:96)"} []
[2014-08-24 00:54:53] app.ERROR: No route found for "GET /v1/" [] []
[2014-08-24 00:54:53] app.ERROR: #0 [internal function]: Symfony\Component\HttpKernel\EventListener\RouterListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\EventDispatcher)) #1 /home/malcolm/silox/vendor/symfony/event-dispatcher/Symfony/Component/EventDispatcher/EventDispatcher.php(164): call_user_func(Array, Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\EventDispatcher)) #2 /home/malcolm/silox/vendor/symfony/event-dispatcher/Symfony/Component/EventDispatcher/EventDispatcher.php(53): Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #3 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(126): Symfony\Component\EventDispatcher\EventDispatcher->dispatch('kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #4 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(66): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #5 /home/malcolm/silox/vendor/silex/silex/src/Silex/Application.php(538): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #6 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(466): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #7 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(429): Symfony\Component\HttpKernel\HttpCache\HttpCache->forward(Object(Symfony\Component\HttpFoundation\Request), true) #8 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(329): Symfony\Component\HttpKernel\HttpCache\HttpCache->fetch(Object(Symfony\Component\HttpFoundation\Request), true) #9 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(193): Symfony\Component\HttpKernel\HttpCache\HttpCache->lookup(Object(Symfony\Component\HttpFoundation\Request), true) #10 /home/malcolm/silox/vendor/silex/silex/src/Silex/HttpCache.php(35): Symfony\Component\HttpKernel\HttpCache\HttpCache->handle(Object(Symfony\Component\HttpFoundation\Request)) #11 /home/malcolm/silox/web/index.php(10): Silex\HttpCache->run() #12 {main} [] []
[2014-08-24 00:55:22] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET //v1/" (uncaught exception) at /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 145 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET //v1/\" at /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php:145, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at /home/malcolm/silox/vendor/symfony/routing/Symfony/Component/Routing/Matcher/UrlMatcher.php:96)"} []
[2014-08-24 00:55:22] app.ERROR: No route found for "GET //v1/" [] []
[2014-08-24 00:55:22] app.ERROR: #0 [internal function]: Symfony\Component\HttpKernel\EventListener\RouterListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\EventDispatcher)) #1 /home/malcolm/silox/vendor/symfony/event-dispatcher/Symfony/Component/EventDispatcher/EventDispatcher.php(164): call_user_func(Array, Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\EventDispatcher)) #2 /home/malcolm/silox/vendor/symfony/event-dispatcher/Symfony/Component/EventDispatcher/EventDispatcher.php(53): Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #3 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(126): Symfony\Component\EventDispatcher\EventDispatcher->dispatch('kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #4 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(66): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #5 /home/malcolm/silox/vendor/silex/silex/src/Silex/Application.php(538): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #6 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(466): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #7 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(429): Symfony\Component\HttpKernel\HttpCache\HttpCache->forward(Object(Symfony\Component\HttpFoundation\Request), true) #8 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(329): Symfony\Component\HttpKernel\HttpCache\HttpCache->fetch(Object(Symfony\Component\HttpFoundation\Request), true) #9 /home/malcolm/silox/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(193): Symfony\Component\HttpKernel\HttpCache\HttpCache->lookup(Object(Symfony\Component\HttpFoundation\Request), true) #10 /home/malcolm/silox/vendor/silex/silex/src/Silex/HttpCache.php(35): Symfony\Component\HttpKernel\HttpCache\HttpCache->handle(Object(Symfony\Component\HttpFoundation\Request)) #11 /home/malcolm/silox/web/index.php(10): Silex\HttpCache->run() #12 {main} [] []
